src/components/ActorList.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div>
+    <div class="d-flex flex-row container" v-if="!loading">
+      <div class="d-flex flex-row">
+          <b-list-group horizontal class="flex-sm-wrap">
+              <b-list-group-item
+                  class="w-auto m-2 p-0 border-0 d-flex justify-content-center align-items-start rounded-circle card"
+                  v-for="actor in actors"
+                  :key="actor.id"
+              >
+                  <actor-card
+                      :name="actor.name"
+                      :knownForDepartment="actor.known_for_department || ''"
+                      :popularity="actor.popularity"
+                      :profilePath="actor.profile_path || ''"
+                      :character="actor.character"
+                      :actorId="actor.id"
+                      @getActorId="getActorDetails"
+                  >
+                  </actor-card>
+              </b-list-group-item>
+          </b-list-group>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import ActorCard from "@/components/ActorCard";
+import { mapState } from "vuex";
+
+export default {
+  name: "ActorList",
+  components: { ActorCard },
+  data() {
+    return {
+      loading: false,
+    }
+  },
+  methods: {
+    async getActorDetails(actorId) {
+      await this.$store.dispatch("getActorDetails", actorId);
```

Comment:
Same comment as above.

Additionally, this async-await pair doesn't do anything. The `getActorDetails` method (the outer one) is unnecessary.